src/aurora/actionscript/variable.cpp

```diff
@@ -25,6 +25,8 @@
 #include "src/aurora/actionscript/variable.h"
 #include "src/aurora/actionscript/object.h"
 #include "src/aurora/actionscript/function.h"
+#include "variable.h"
```

Comment:
Full paths, please.

Also, src/aurora/actionscript/variable.h is already included in line 25.